Sources/XcbeautifyLib/Pattern.swift

```diff
@@ -326,6 +326,12 @@ enum Pattern: String {
     /// $1 = whole warning
     case willNotBeCodeSigned = #"^(.* will not be code signed because .*)$"#
 
+    /// Regular expresion captured groups:
+    /// $1 = duplicate key
+    /// $2 = value kept
+    /// $3 = value ignored
+    case duplicateLocalizedStringKey = #"^[\d\s-:]+ --- WARNING: Key "(.*)" used with multiple values. Value "(.*)" kept. Value "(.*)" ignored.$"#
```

Comment:
Do we need to capture these values since we only remove the `--- WARNING:` prefix?

Reply:
I guess they could be ignored and just re-out the entire message minus the prefix. I'll update that.

Reply:
Updated in latest commit.